app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+# app/controllers/posts_controller.rb
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
```

Comment:
- [ ] [optional] @NoelLincoln Great work adding all the required action methods, however, it would be nice to remove the functionality and leave them as blank methods for now, this will prevent them from generating errors as shown below due to lack of data in your database.
![Uploading image.png…]()